libopae/src/reconf.c

```diff
@@ -170,33 +169,26 @@ static fpga_result port_busy(fpga_handle handle)
 	//       slot being reconfigured
 	result = fpgaEnumerate(&props, 1, &token, 1, &matches);
 	if (result != FPGA_OK) {
-		FPGA_ERR("Error enumerating while reconfiguring");
+		FPGA_ERR("Error enumerating for accelerator to reconfigure");
 		goto free_props;
 	}
 
-	if (0 == matches) {
-		FPGA_ERR("No accelerators found while reconfiguring.");
+	if (matches == 0) {
+		FPGA_ERR("No accelerator found to reconfigure");
```

Comment:
Just wondering - if we can't find an accelerator whose parent is our FME, doesn't that mean we should go ahead with the reconfiguration? After all, if _we_ can't find the accelerator, probably no one else has it.

Reply:
Not sure. What happens in the case of virtualization?

Reply:
Good point. If we pass through the VF to a VM, we won't find an accelerator either. Would we want to fail reconfiguration in that case, if "--force" is not given?

Reply:
Yes. This check is only performed if `--force` (or `FPGA_RECONF_FORCE`) is not in the flags.